cli/src/pcluster/resources/imagebuilder/parallelcluster_validate.yaml

```diff
@@ -305,7 +305,7 @@ phases:
                   rpm -qa | grep intel-mpi
                   [[ $? -ne 0 ]] && echo "Check intel-mpi rpm failed" && exit 1
                 else
-                  dpkg -l | grep libfabric && dpkg -l | grep 'efa '
```

Comment:
modinfo efa is good to verify that the kernel driver is installed but does not guarantees efa libs are. One way to check for full installation is to probably verify the existence of /opt/amazon/efa or /opt/amazon/efa_installed_packages

Reply:
I'm wondering if we want to verify if the installed version is the same defined in the recipe, or just that it's installed

Reply:
if efa is not installed by our recipe, we can't check the version defined in recipe